client/src/components/App/Content/UsersManagement/useUsersManagement.ts

```diff
@@ -200,6 +200,22 @@ const useUsersManagement = ({ page, rowsPerPage, cellNameSort, setPage }: useUse
 
     const handleCloseDialog = () => setUserDialog({ isOpen: false, info: {} })
 
+    const setUserActivityStatus = (isActive: boolean, userId: string) : Promise<any> => {
+        const setUpdateActivityStatusLogger = logger.setup('GraphQL request to the DB');
+        setUpdateActivityStatusLogger.info('started is user active updating', Severity.LOW);
+        return axios.post('users/updateIsUserActiveById', {
+            isActive,
+            userId
+        }).then((result) => {
+            if(result.data)
+                fetchUsers();
```

Comment:
Why fetching the users again and not updating the fetches users in the client

Reply:
to keep single source of truth